### ISSUE TITLE
Abnormality of mental imagery

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#11443]
**Preferred term label:**
Abnormality of mental imagery

**Definition (free text, please give PubMed ID)**
A deviation from the typical ability to voluntarily generate, maintain, or experience mental images representing sensory properties of objects or events in their absence / without a direct external stimulus. (PMID: 26412097; 23383863)

**Parent term (use hpo.jax.org/app)**
Cognitive impairment 
HP:0100543

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
@bcoleman96 - thoughts on this NTR?

**Comment by bcoleman96** [comment]
This and the linked terms look good to me. I wish there was a more specific parent but I don't think one exists.

**Comment by pnrobinson** [comment]
@jagcsun added as HP:0020344, thanks! (Child of Abnormal cognitive process)
